runelite-client/src/main/java/net/runelite/client/plugins/metronome/MetronomePlugin.java

```diff
@@ -57,6 +57,11 @@ MetronomePluginConfiguration provideConfig(ConfigManager configManager)
 	{
 		return configManager.getConfig(MetronomePluginConfiguration.class);
 	}
+	@Override
+	protected void startUp()
```

Comment:
Generally, resetting plugin state should happen when plugins shut down.
```suggestion
	protected void shutDown()
```